video/out/vulkan/context.h

```diff
@@ -7,8 +7,11 @@ struct ra_vk_ctx_params {
     // See ra_swapchain_fns.get_vsync.
     void (*get_vsync)(struct ra_ctx *ctx, struct vo_vsync_info *info);
 
+    // In case something special needs to be done when starting a frame.
+    bool (*start_frame)(struct ra_ctx *ctx);
```

Comment:
Maybe this is confusing. Vulkan doesn't have an external swapchain. I could have added it to the context, but I didn't think it really made sense with the libplacebo backend. So instead I just made this an additional parameter.